modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -87,7 +88,8 @@ export default class TileLayer extends CompositeLayer {
         onTileLoad: this._onTileLoad.bind(this),
         onTileError: this._onTileError.bind(this),
         onTileUnload: this._onTileUnload.bind(this),
-        maxRequests
+        maxRequests,
+        modelMatrix
```

Comment:
This doesn't account for changes in the `modelMatrix` prop, which could potentially happen on every render. It should be passed into `getTileIndices` instead of the constructor.